README.md

```diff
@@ -1 +1,2 @@
-# GaussianProcessRegression
\ No newline at end of file
+# GaussianProcessRegression
+Random Fourier Features accelerated Gaussian Process Regressor
```

Comment:
Provide an overview of the bundle.  Specifically, The purpose and constraints of the bundleshould be discussed, and the available "public" modules should be defined and described.  For example:

GPRI is the main module that allows fitting of models and making predictions based on the fitted model.
Types provides ...
Usage examples can be found in the Test folder
etc.

Need to acknowledge the python module that we built it from.

Installation instructions:

Do we have any installation dependencies?
ecl bundle install ....